assets/aws/files/bin/teleport-ssm-publish-tokens

```diff
@@ -10,6 +10,15 @@ set -o pipefail
 . /etc/teleport.d/conf
 
 TCTL=/usr/local/bin/tctl
+# get the tctl version
+# database service tokens are only supported in Teleport 6+
+TCTL_VERSION=$(${TCTL} version | cut -dv -f2 | cut -d' ' -f1)
+TCTL_MAJOR_VERSION=$(echo ${TCTL_VERSION} | cut -d'.' -f1)
+DATABASE_TOKEN_ENABLED=false
+if [ ${TCTL_MAJOR_VERSION} -ge 6 ]; then
```

Comment:
what happens if `TCTL_MAJOR_VERSION` is an empty string?

Reply:
Theoretically due to `set -e` then the script will exit earlier if `tctl version` or any of the `cut` commands fail. I will see whether I can make it a bit more resilient though.

Reply:
It's possible that our version format will change at some point, but not enough to fail any of the commands above

Reply:
Addressed in dbeee9a3005e511ba26f069b493868d718d85df2